docs/en/setup/backend/backend-mysql-monitoring.md

```diff
@@ -0,0 +1,40 @@
+# MYSQL monitoring 
+SkyWalking leverages prometheus/mysqld_exporter for collecting metrics data from MYSQL. It leverages OpenTelemetry Collector to transfer the metrics to
+[OpenTelemetry receiver](opentelemetry-receiver.md) and into the [Meter System](./../../concepts-and-designs/meter.md). 
+
+## Data flow
+1. mysqld_exporter collect metrics data from MYSQL.
+2. OpenTelemetry Collector fetches metrics from mysqld_exporter via Prometheus Receiver and pushes metrics to SkyWalking OAP Server via the OpenCensus GRPC Exporter.
+3. The SkyWalking OAP Server parses the expression with [MAL](../../concepts-and-designs/mal.md) to filter/calculate/aggregate and store the results. 
+
+## Setup 
+1. Setup [mysqld_exporter](https://github.com/prometheus/mysqld_exporter#using-docker).
+2. Set up [OpenTelemetry Collector ](https://opentelemetry.io/docs/collector/getting-started/#docker). For details on Prometheus Receiver in OpenTelemetry Collector, refer to [here](https://github.com/prometheus/prometheus/blob/main/documentation/examples/prometheus-docker.yml). 
```

Comment:
```suggestion
2. Set up [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/getting-started/#docker). For details on Prometheus Receiver in OpenTelemetry Collector, refer to [here](https://github.com/prometheus/prometheus/blob/main/documentation/examples/prometheus-docker.yml). 
```

Reply:
I think you missed where and how open telemetry collector should be set.
A copy of the setting file is recommended, as we can't control other provide link is never changed, and you will need that find in required e2e test

Reply:
I don't fully understand.
Should I paste my Opentelemetry Collector config file into this doc or put it somewhere else and refer to it in the document?

Reply:
You need to add an e2e test, then you need to use this configuration file.
You could link that file.

Reply:
I understand.